dipy/viz/horizon/tab/slice.py

```diff
@@ -325,9 +355,32 @@ def __update_slice_visibility_z(self):
         self.__slider_slice_z.set_visibility(self.__slice_visibility_z)
         self.__actors[2].SetVisibility(self.__slice_visibility_z)
 
-    def build(self, tab_id, tab_ui):
+    def update_slices(self, x_slice, y_slice, z_slice):
+        """
+        Updates slicer positions
+
+        Parameters
+        ----------
+        x_slice: float
+            x-value where the slicer should be placed
+        y_slice: float
+            y-value where the slicer should be placed
+        z_slice: float
+            z-value where the slicer should be placed
+        """
+        if not self.__slider_slice_x.value == x_slice:
+            self.__slider_slice_x.value = x_slice
+
+        if not self.__slider_slice_y.value == y_slice:
+            self.__slider_slice_y.value = y_slice
+
+        if not self.__slider_slice_z.value == z_slice:
+            self.__slider_slice_z.value = z_slice
+
+    def build(self, tab_id, tab_ui, tab_manager):
         self.__tab_id = tab_id
         self.__tab_ui = tab_ui
+        SlicesTab.tab_manager = tab_manager
```

Comment:
This seems very helpful. Make sure that it does not create any issues with other things e.g. other panels etc.

Reply:
No, it will not I am managing it in all the tabs types.